python/xoscar/backends/communication/socket.py

```diff
@@ -258,12 +259,29 @@ async def connect(
         return SocketClient(local_address, dest_address, channel)
 
 
-TEMPDIR = tempfile.gettempdir()
+def _get_or_create_default_unix_socket_dir():
+    import sys
+
+    if sys.platform.startswith("win"):
+        unix_socket_dir = XOSCAR_UNIX_SOCKET_DIR_WIN
```

Comment:
Why does this line not covered by windows CI?

Reply:
This is unexpected. I'll double check it on a Windows PC.